buildconfig/Jenkins/Conda/main-pipeline.jenkinsfile

```diff
@@ -0,0 +1,114 @@
+def git_clone(){
+    git url: 'https://github.com/mantidproject/mantid.git',
+        branch: 'main_pipeline_prototype'
+}
+
+def build_and_test_linux() {
+    git_clone()
+    sh "$WORKSPACE/buildconfig/Jenkins/Conda/conda-buildscript $WORKSPACE linux-ci --enable-systemtests --enable-doctests --clean-build --clean-external-projects"
```

Comment:
Since we are directly invoking the bash shell in the scripts, and rely on bash-isms can we change these from `sh` -> `bash` to make it clear